moveit_core/online_signal_smoothing/CMakeLists.txt

```diff
@@ -45,18 +45,47 @@ ament_target_dependencies(moveit_butterworth_filter
   pluginlib
 )
 
+add_library(moveit_acceleration_filter SHARED
+  src/acceleration_filter.cpp
+)
+generate_export_header(moveit_acceleration_filter)
+target_include_directories(moveit_acceleration_filter PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
+)
+set_target_properties(moveit_acceleration_filter PROPERTIES VERSION
+  "${${PROJECT_NAME}_VERSION}"
+)
+
+generate_parameter_library(moveit_acceleration_filter_parameters src/acceleration_filter_parameters.yaml)
+
+target_link_libraries(moveit_acceleration_filter
+  moveit_acceleration_filter_parameters
+  moveit_robot_state
+  moveit_smoothing_base
+  osqp::osqp
+)
+ament_target_dependencies(moveit_acceleration_filter
+  srdfdom  # include dependency from moveit_robot_model
+)
```

Comment:
I think you need to add `pluginlib` here, which may be why this is failing:
https://github.com/ros-planning/moveit2/actions/runs/7576002880/job/20633851536?pr=2651#step:12:251